src/main/java/com/example/fooddelivery/service/impl/DefaultOrderService.java

```diff
@@ -0,0 +1,83 @@
+package com.example.fooddelivery.service.impl;
+
+import com.example.fooddelivery.dto.*;
+import com.example.fooddelivery.enums.DeliveryStatus;
+import com.example.fooddelivery.exceptions.EntityNotFoundException;
+import com.example.fooddelivery.model.Address;
+import com.example.fooddelivery.model.Order;
+import com.example.fooddelivery.model.OrderEntry;
+import com.example.fooddelivery.model.User;
+import com.example.fooddelivery.repository.OrderRepository;
+import com.example.fooddelivery.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service
+public class DefaultOrderService implements OrderService {
```

Comment:
Wouldn't it be better if you get models inside service layer, and after that convert them to dto's inside facade layer?

Reply:
I don't know. Let's find some useful links.